Hazel/src/Hazel.h

```diff
@@ -2,6 +2,8 @@
 
 // For use by Hazel applications
 
+#include "Hazel/Core/Core.h"
```

Comment:
Yes, it is indeed called `Base.h`. However, since we use it alot, shouldn't that be mensioned in a video instead? Tbh, I didn't even think about this one. Maybe that one should have a dedicated PR?

Reply:
I just took a look at Hazel-dev to see what plans Cherno could have with Core.h and I've decided that I'll leave the maintenance to him, as it is more than "just renaming the file". So I'll mark this as resolved as I won't implement this.